mkdocs.yml

```diff
@@ -131,6 +131,7 @@ nav:
   - Home: README.md
   - Getting Started:
       - Introduction to Ansible and AVD: docs/getting-started/intro-to-ansible-and-avd.md
+      - Ansible Automation Platform and AVD: docs/getting-started/avd-aap.md
```

Comment:
Tiny thing - the page's title is "AVD and Ansible Automation Platform" whereas the index says "Ansible Automation Platform and AVD". I feel the page's title is better and the index could use it

Reply:
Fair enough, easy change :)